### PR TITLE
Add a basic admin action to add default geo filters

### PR DESCRIPTION
**Pull request by ericholscher**

### CODE REVIEW

**Comment by agjohnson** [comment]
Would it make sense to set this as a default state on the filter model instead?

**Comment by ericholscher** [comment]
I think it's probably better to just make it easy to add, not make it the default. I think we can serve stuff to those geo's in general, but it's nice to have an easy way to exclude them.

**Comment by agjohnson** [comment]
I just meant in terms of producing this on the admin. I think the general case will be all locations as well. If it was set as the model default, the countries should show up on the list automatically, without a secondary action. This might only make sense if it will be a common filter, of course.

File: readthedocs/donate/admin.py [hunk 1]
```diff
@@ -1,8 +1,20 @@
...
+DEFAULT_EXCLUDES = ['BD', 'CN', 'TF', 'GT', 'IN', 'ID',
+                    'IR', 'PK', 'PH', 'RU', 'TW', 'TH', 'TR', 'UA', 'VN']
+
+
+def set_default_countries(modeladmin, request, queryset):
+    for project in queryset:
+        filter = project.geo_filters.create(filter_type='exclude')
+        for country in Country.objects.filter(country__in=DEFAULT_EXCLUDES):
+            filter.countries.add(country)
+set_default_countries.short_description = "Add default exclude countries to this Promo"
```
Comment (agjohnson) [inline readthedocs/donate/admin.py]: This could belong in the `GeoFilterAdmin` class below, keeping everything encapsulated in the class.

**Pull request merged**